Neos.ContentRepository/Classes/Security/Authorization/Privilege/Node/AbstractNodePrivilege.php

```diff
@@ -68,15 +82,26 @@ public function initialize()
         $this->initializeMethodPrivilege();
     }
 
+    /**
+     * @return void
+     */
+    protected function buildCacheEntryIdentifier()
+    {
+        $this->cacheEntryIdentifier = md5($this->privilegeTarget->getIdentifier() . '__methodPrivilege' . '|' . $this->buildMethodPrivilegeMatcher());
```

Comment:
any reason not to use the existing `getPrivilegeTargetIdentifier` instead of `$this->privilegeTarget->getIdentifier()` method here?

Reply:
@kitsunet Can you clarify this? Is that needed due to lazy initialisation or something?

Reply:
by testing look like calling `getPrivilegeTargetIdentifier` doesn't make any difference.

Reply:
I think there were edge cases in which using `getPrivilegeTargetIdentifier` triggered additional initialisations that were not really needed.